src/xdocs/index.xml.vm

```diff
@@ -551,7 +551,7 @@
         </div>
       </subsection>
 
-      <subsection name="Inactive / Old Tools" id="Related_Tools_Inactive_Old_Tools">
+      <subsection name="Inactive / Old Tools" id="Related_Tools_Inactive_/_Old_Tools">
         <div class="wrapper">
```

Comment:
Such id will not work, let's change `/`  to `or`

Reply:
Done.